library/src/main/java/com/google/android/exoplayer/extractor/mp4/AtomParsers.java

```diff
@@ -114,10 +113,10 @@ public static TrackSampleTable parseStbl(Track track, Atom.ContainerAtom stblAto
     Atom.LeafAtom cttsAtom = stblAtom.getLeafAtomOfType(Atom.TYPE_ctts);
     ParsableByteArray ctts = cttsAtom != null ? cttsAtom.data : null;
 
+    int fixedSampleSize = sampleSizeBox.getFixedSampleSize();
```

Comment:
there's a few usages of fixedSampleSize outside of line 197 that don't perfect reconcile. I added a new method to the samplesizebox interface that allows us to tell if a sample is fixed size without needing to expose the size itself (until it's used in the fixed size rechunking code below).